what4/test/PrinterTests.hs

```diff
@@ -0,0 +1,63 @@
+
+{-# LANGUAGE DataKinds #-}
+
+import Test.Tasty
+import Test.Tasty.HUnit
+import What4.InterpretedFloatingPoint(FloatInfoRepr (HalfFloatRepr, SingleFloatRepr, DoubleFloatRepr, QuadFloatRepr, X86_80FloatRepr, DoubleDoubleFloatRepr))
+import Prettyprinter
+import What4.BaseTypes (StringInfoRepr(UnicodeRepr, Char16Repr, Char8Repr), FloatPrecisionRepr (FloatingPointPrecisionRepr), NatRepr)
+import Data.Parameterized (knownNat)
```

Comment:
A somewhat arbitrary stylistic choice, but I'd prefer to alphabetize these (or at least group them so that the `What4` imports are next to each other).